pkg/cmd/registry/artifact/state/state.go

```diff
@@ -0,0 +1,121 @@
+package state
+
+import (
+	"context"
+	"errors"
+
+	"github.com/redhat-developer/app-services-cli/pkg/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/iostreams"
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/serviceregistry/registryinstanceerror"
+	"github.com/spf13/cobra"
+
+	"github.com/redhat-developer/app-services-cli/internal/config"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/factory"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/registry/artifact/util"
+	"github.com/redhat-developer/app-services-cli/pkg/logging"
+	registryinstanceclient "github.com/redhat-developer/app-services-sdk-go/registryinstance/apiv1internal/client"
+)
+
+type SetOptions struct {
```

Comment:
I hate generic names :)

Reply:
They are not generic because the name of the package is inclusive to the name.

`state.Options`. In fact we should make all command "options" private by default. They were originally public with the anticipation of testing, but since we did not there is no longer any need.